elasticsearch/aws_plugin.sls

```diff
@@ -2,8 +2,9 @@ install_aws:
   cmd:
   - run
   - user: root
-  - name: '/usr/share/elasticsearch/bin/plugin install cloud-aws'
+  - name: '/usr/share/elasticsearch/bin/plugin install -b cloud-aws ; /sbin/service elasticsearch restart'
```

Comment:
If we do need the restart I think it'd be better to have a restart state using `service` and have it follow this state so it's more clear that installing this plugin will restart the service.